src/lib.rs

```diff
@@ -111,15 +120,15 @@ impl<T: Clone> Clone for Vc<T> {
     }
 }
 
-impl<T> Default for Vc<T> {
+impl<T, const R: usize> Default for CustomVc<T, R> {
```

Comment:
Would you mind checking that `Vc::default()` still works? I worry about the type inference not realizing that the type alias setting `R`, and as a result complaining that the call is ambiguous.

Reply:
I added a new Test case for this [which can be found here](https://github.com/jonhoo/atone/pull/15/commits/357396f7b9f290d16d8a702df105ffd77f0a2ed2#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R2598-R2605) and that seems to work just fine for me, so I think this is not a problem